apps/functions-subscription/src/adapters/azure/functions/middleware.ts

```diff
@@ -51,7 +63,7 @@ export const parseHeaderParameter =
       ),
     );
 
-type AllowedGroup = 'ApiTrialUser' | 'ApiTrialManager';
+type AllowedGroup = 'ApiTrialUser' | 'ApiTrialManager' | 'ApiTrialSupport';
```

Comment:
**issue**: this change is in a [different PR](https://github.com/pagopa/trial-system/pull/256/files#diff-54d80969596ade2ef3206d66ed37285303a34f0ab736458d9b9e7089c5166de2R54)

Reply:
Resolved in 830a067a8470df94ce7f2ed67c6bc641097ad847